### PR TITLE
README: clarify licensing

### PR DESCRIPTION
**Pull request by ForNeVeR**
Based on discussion with @rexim and @Minoru. We're saying that we're ready to consider any licenses that are not (yet) in the list.

### CODE REVIEW

**Review by rexim — APPROVED** [review]

**Comment by rexim** [comment]
@ForNeVeR thanks!

**Comment by ForNeVeR** [comment]
Please note that personally I support that proposal, but I'd like to hear feedback from @Minoru.

**Comment by rexim** [comment]
@Minoru feel free to leave your feedback here or as a separate PR.

File: README.md [hunk 1]
```diff
@@ -95,13 +95,16 @@ process of the projects they lead.
 
 ## Project Licensing Requirement ##
 
-All of the projects of [@morganey-lang] community are required to
-be licensed under one of the following licenses:
+Here's a list of licenses recommended by [@morganey-lang] to use in the
+community projects:
```
Comment (Minoru) [inline README.md:99]: It should be either "recommended for use" or "recommended to be used".

The whole sentence can also be rewritten like that: "We recommend using one of the following licenses for [@morganey-lang] community projects". This wording makes it clear that you're recommending licenses, not the list itself.

**Pull request merged**